cvat-core/README.md

```diff
@@ -10,7 +10,9 @@ It contains the core logic of the Computer Vision Annotation Tool.
 - Dependencies installation
 
 ```bash
-yarn install --frozen-lockfile
+corepack enable yarn
+yarn set version berry
```

Comment:
I am not sure we need this command here as we already have specified version in package.json